v03_pipeline/lib/tasks/write_remapped_and_subsetted_callset.py

```diff
@@ -69,6 +72,7 @@ def requires(self) -> list[luigi.Task]:
                 *requirements,
                 self.clone(WriteRelatednessCheckTableTask),
                 self.clone(WriteSexCheckTableTask),
+                self.clone(WriteRelatednessCheckTsvTask),
```

Comment:
Any reason to not just delete `WriteRelatednessCheckTableTask`?

Reply:
didn't think to but it is definitely the move